Extension/src/LanguageServer/configurations.ts

```diff
@@ -1574,7 +1577,7 @@ export class CppProperties {
         }
 
         if (resolvedCompilerPath === undefined) {
-            resolvedCompilerPath = this.resolvePath(config.compilerPath);
+            resolvedCompilerPath = this.resolvePath(config.compilerPath ?? undefined);
```

Comment:
The 1st arg of resolvePath can just add a `| null` and then you can remove the `?? undefined` here.